Packs/ThreatGrid/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,40 @@
+#### Integrations
+
+##### Cisco Threat Grid
+- Update all integration commands
```

Comment:
What does the update include? This should let the user know what to expect.
See [here](https://xsoar.pan.dev/docs/documentation/release-notes#what-should-be-logged)